src/cascadia/inc/cppwinrt_utils.h

```diff
@@ -17,6 +17,15 @@ Revision History:
 
 #pragma once
 
+template<>
+struct fmt::formatter<winrt::hstring, wchar_t> : fmt::formatter<fmt::wstring_view, wchar_t>
+{
+    auto format(const winrt::hstring& str, auto& ctx)
+    {
+        return fmt::formatter<fmt::wstring_view, wchar_t>::format({ str.data(), str.size() }, ctx);
+    }
+};

```

Comment:
It seems like we don't have any `fmt::format` call that uses `FMT_COMPILE` on `hstring` arguments, or any `fmt::format` that uses >1 `hstring` argument. The above one seems to be a first.

...because it appears as if right now we implicitly cast all `hstring` arguments to C-strings during formatting. This new custom formatter ensures they're treated like string-views (and it allows the above code to compile).

Reply:
_i love this_. does it make the transition to 10.1.1 any easier?